experimental/dds/tree2/src/feature-libraries/sequence-field/moveEffectTable.ts

```diff
@@ -53,7 +53,7 @@ export interface MovePartition<TNodeChange> {
 	// Undefined means the partition is the same size as the input.
 	count?: number;
 	replaceWith?: Mark<TNodeChange>[];
-	modifyAfter?: TNodeChange;
+	modifyAfter?: [TNodeChange, RevisionTag | undefined];
```

Comment:
Is there an advantage to making this a tuple instead of an object with named fields?